README.md

```diff
@@ -1,16 +1,16 @@
-gocql
+Apache Cassandra GoCQL Driver
 =====
 
-[![Join the chat at https://gitter.im/gocql/gocql](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/gocql/gocql?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-![go build](https://github.com/gocql/gocql/actions/workflows/main.yml/badge.svg)
-[![GoDoc](https://godoc.org/github.com/gocql/gocql?status.svg)](https://godoc.org/github.com/gocql/gocql)
+[!Join the chat at https://the-asf.slack.com/archives/C05LPRVNZV1](https://the-asf.slack.com/archives/C05LPRVNZV1)
+![go build](https://github.com/apache/cassandra-gocql-driver/actions/workflows/main.yml/badge.svg)
+[![GoDoc](https://godoc.org/github.com/apache/cassandra-gocql-driver?status.svg)](https://godoc.org/github.com/apache/cassandra-gocql-driver)
```

Comment:
This documentation link should point to the same URL as the one below (https://godoc.org/github.com/gocql/gocql)

Resp. we could replace all godoc link to pkg.go.dev, but that should perhaps be a separate commit/MR.